### PR TITLE
Split cereal into cereal/msgq

### PR DESCRIPTION
**Pull request by haraschax**

### CODE REVIEW

**Comment by adeebshihadeh** [comment]
Jenkins checkout should be fixed if you rebase.

**Comment by haraschax** [comment]
Should be good to merge now, bunch of cleanups I will get started on once merged.

File: cereal/logger/logger.h [hunk 1]
```diff
@@ -0,0 +1,21 @@
+#pragma once
+
```
Comment (adeebshihadeh) [inline cereal/logger/logger.h:2]: this should be removed. it's used to pass in openpliot's logger to submodules

File: .gitignore [hunk 2]
```diff
@@ -41,6 +41,9 @@ compare_runtime*.html
 
 persist
 selfdrive/pandad/pandad
+board/obj/
```
Comment (adeebshihadeh) [inline .gitignore:44]: not sure what this is for? bad merge?

**Review by adeebshihadeh — COMMENTED** [review]
A bunch of the files in `cereal/` should be removed or moved to `msgq/`

**Pull request merged**